year-pages/2019-conferences-list.md

```diff
@@ -0,0 +1,5 @@
+## 2019
+
+| Conference | Date | Venue | Price | register_before | scholarship |
```

Comment:
Love the scholarship column! :sparkling_heart: 

I would prefer the price column to not be included because it depends on a lot of things (example, are you adding a workshop, etc) and also changes as the conference day gets closer.

Also, I would remove the register_before column because it's usually until the day before the conference.

Can you add a description column though? Just like it's there for the 2018 conferences. Thanks!

Reply:
one tiny nit: s/scholarship/Scholarship (capital first letter in the heading)